librarian_packages/hera_librarian/tests/test_utils.py

```diff
@@ -0,0 +1,176 @@
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2019 the HERA Collaboration
+# Licensed under the 2-clause BSD License
+
+"""Test code in hera_librarian/utils.py
+
+"""
+
+import pytest
+import os
+import six
+import sys
+from contextlib import contextmanager
+from hera_librarian import utils
+
+# import test data attributes from __init__.py
+from . import ALL_FILES, obsids, filetypes, md5sums, pathsizes
+
+
+def test_get_type_from_path():
+    """Test type checking from path"""
+    path = "/some/long/file.name.txt"
+    assert utils.get_type_from_path(path) == "txt"
+
+    return
+
+
+def test_get_pol_from_path():
+    """Test polarization extraction from filename"""
+    filename = "zen.2458000.12345.xx.uvh5"
+    assert utils.get_pol_from_path(filename) == "xx"
+
+    filename = "zen.2458000.12345.uvh5"
+    assert utils.get_pol_from_path(filename) is None
+
+    return
+
+
+@ALL_FILES
+def test_get_obsid_from_path(datafiles):
+    """Test extracting obsid values from datasets"""
+    obsid_uvh5 = 1225829886
+    obsid_miriad = 1192201262
+    obsids = [obsid_uvh5, obsid_miriad]
```

Comment:
Can this just use the `obsids` variable imported from the `__init__` file?

Reply:
Yes, good point. Changed.